src/hotspot/os/posix/signals_posix.cpp

```diff
@@ -47,6 +47,14 @@
 // signal handling (except suspend/resume)
 // suspend/resume
 
+// Glibc on Linux uses the SA_RESTORER flag to indicate
+// the use of a "signal trampoline". We have no interest
+// in this flag and need to ignore it when checking our
+// own flag settings.
+// Note: SA_RESTORER is not exposed through signal.h so we
```

Comment:
Should this comment mention `SA_RESTORER` or `SA_RESTORER_FLAG_MASK`?

Reply:
I'll tweak the comment before pushing.